builtin/providers/aws/resource_aws_ses_receipt_rule_set.go

```diff
@@ -0,0 +1,101 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesReceiptRuleSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesReceiptRuleSetCreate,
+		Read:   resourceAwsSesReceiptRuleSetRead,
+		Delete: resourceAwsSesReceiptRuleSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"rule_set_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceAwsSesReceiptRuleSetCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	ruleSetName := d.Get("rule_set_name").(string)
+
+	createOpts := &ses.CreateReceiptRuleSetInput{
+		RuleSetName: aws.String(ruleSetName),
+	}
+
+	_, err := conn.CreateReceiptRuleSet(createOpts)
+	if err != nil {
+		return fmt.Errorf("Error creating SES rule set: %s", err)
+	}
+
+	d.SetId(ruleSetName)
+
+	return resourceAwsSesReceiptRuleSetRead(d, meta)
+}
+
+func resourceAwsSesReceiptRuleSetRead(d *schema.ResourceData, meta interface{}) error {
+	ruleSetExists, err := findRuleSet(d.Id(), nil, meta)
+
+	if !ruleSetExists {
+		d.SetId("")
```

Comment:
I think a Log message should be added here to note that it was removed from state